app/src/main/res/values-fr/strings.xml

```diff
@@ -3,6 +3,9 @@
     <!-- Shortcut strings -->
     <string name="scut_deleted_pc">PC supprimé</string>
     <string name="scut_not_paired">PC non appairé</string>
+    <string name="scut_pc_not_found">PC non trouvé</string>
+    <string name="scut_invalid_uuid">Le PC fourni n'est pas valide</string>
+    <string name="scut_invalid_app_id">L'application fournie n'est pas valide</string>
```

Comment:
`'` characters need a `\` before them

Reply:
thank you for the remark I had not seen the one I change that